tt_eager/tt_dnn/op_library/bmm/multi_core_reuse_mcast_2d_optimized/bmm_op_multi_core_reuse_mcast_2d_optimized.cpp

```diff
@@ -344,6 +348,13 @@ operation::ProgramWithCallbacks create_program_mcast_in0_in1(
     if (fp32_dest_acc_en) {
         mm_kernel_defines["FP32_DEST_ACC_EN"] = "1";
     }
+    if (in0_height_sharded) {
+        mm_kernel_in0_sender_defines["IN0_SHARDED"] = "1";
+    }
+    if (in1_is_sharded) {
+        mm_kernel_in1_sender_writer_defines["IN1_SHARDED"] = "1";
+    }
```

Comment:
Can we rename our flags to be more explicit, like `IN0_HEIGHT_SHARDED` and `IN1_WIDTH_SHARDED`.

Reply:
I don't think so since these #ifdefs are used in matmul1d in different context?
Correct me if I'm wrong.

Reply:
matmul 1D also use the same kernel? I guess it's fine to leave as is